app/views/steps/capital/savings/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= t('steps.capital.caption') %></span>
+
+    <%= govuk_error_summary(@form_object) %>
+
+    <h1 class="govuk-heading-xl">
+      <%= t(".heading.#{@form_object.saving_type}") %>
+    </h1>
+
+    <%= step_form @form_object do |f| %>
+      <%= f.govuk_text_field :provider_name, autocomplete: 'off' %>
+      <%= f.govuk_text_field :sort_code, autocomplete: 'off', inputmode: 'numeric', extra_letter_spacing: true, width: 'one-half' %>
```

Comment:
Is there any way to get rid of that speech bubble on the text field in the screenshot?